tests/js/spec/views/dashboardsV2/utils.spec.tsx

```diff
@@ -141,4 +143,75 @@ describe('Dashboards util', () => {
       );
     });
   });
+
+  describe('getWidgetDiscoverUrl', function () {
+    let widget;
+    beforeEach(() => {
+      widget = {
+        title: 'Test Query',
+        displayType: DisplayType.LINE,
+        widgetType: WidgetType.DISCOVER,
+        interval: '5m',
+        queries: [
+          {
+            name: '',
+            conditions: '',
+            fields: ['count()'],
+            orderby: '',
+          },
+        ],
+      };
+    });
+    it('returns the discover url of the widget query', () => {
+      const url = getWidgetDiscoverUrl(widget, selection, TestStubs.Organization());
+      expect(url).toEqual(
+        '/organizations/org-slug/discover/results/?field=count%28%29&name=Test%20Query&query=&statsPeriod=7d&yAxis=count%28%29'
+      );
+    });
+    it('returns the discover url of a topn widget query', () => {
+      widget = {
+        ...widget,
+        ...{
+          displayType: DisplayType.TOP_N,
+          queries: [
+            {
+              name: '',
+              conditions: 'error.unhandled:true',
+              fields: ['error.type', 'count()'],
+              orderby: '-count',
+            },
+          ],
+        },
+      };
+      const url = getWidgetDiscoverUrl(widget, selection, TestStubs.Organization());
+      expect(url).toEqual(
+        '/organizations/org-slug/discover/results/?display=top5&field=error.type&name=Test%20Query&query=error.unhandled%3Atrue&sort=-count&statsPeriod=7d&yAxis=count%28%29'
+      );
+    });
+  });
+  describe('getWidgetIssueUrl', function () {
+    let widget;
+    beforeEach(() => {
```

Comment:
do we plan to add more tests here?

Reply:
Yes, I plan to add some more tests in this area as a future pr! Let me know if you're ok with this!